cirq-core/cirq/ops/linear_combinations.py

```diff
@@ -469,12 +501,23 @@ def with_qubits(self, *new_qubits: 'cirq.Qid') -> 'PauliSum':
         return PauliSum.from_pauli_strings(new_pauli_strings)
 
     def copy(self) -> 'PauliSum':
+        """Return a copy of this PauliSum.
+
+        Returns: A copy of this PauliSum.
+        """
         factory = type(self)
         return factory(self._linear_dict.copy())
 
     def matrix(self, qubits: Optional[Iterable[raw_types.Qid]] = None) -> np.ndarray:
-        """Reconstructs matrix of self from underlying Pauli operations in
-        computational basis of qubits.
+        """Get the matrix representing this PauliSum.
+
+        Args:
+            qubits: Iterable of qubits, specifying ordering to use
+                in the returned matrix. If none is provided
```

Comment:
"ordering to use in the returned matrix" isn't very clear. Maybe replace with "ordering to determine the computational basis of qubits"

Reply:
Changed.